docs/_documentations/mdt-vsc-firstproject.md

```diff
@@ -1,52 +1,54 @@
 ---
 layout: docs
-title: "Codewind for VS Code"
-description: "Codewind for VS Code"
+title: "Creating your first VS Code Codewind project"
+description: "Creating your first VS Code Codewind project"
 keywords: introducing, introduction, overview, what is, tools, vscode, visual, studio, code, java, microprofile, spring, node, nodejs, node.js, javascript, Codewind for VS Code, tools, view, debug, integrate, open a shell session, toggle auto build, manually build, scope VS Code workspace, disable, enable, delete
 duration: 1 minute
 permalink: mdt-vsc-firstproject
 type: document
 order: 2
 parent: mdt-vsc-overview
 ---
-# Creating your first project
+# Creating your first VS Code Codewind project
 <br/>
-To create your first project, click on the prompt in the **CODEWIND** window
+To create your first project:
+
+1\. Click on the prompt in the **CODEWIND** window.
 
 ![](dist/images/createproject.png)
 
-This will prompt you with a list of project types to select from.
+2\. This prompt gives you a list of project types for you to select. 
```

Comment:
Not keen on `gives`, how about: 
This prompt displays a list of project types. Select the project type that you require from the list.